CONTRIBUTING.md

```diff
@@ -70,6 +70,7 @@ Some packages like `ai` also have more details tests and watch mode, see their `
 We greatly appreciate your pull requests. Here are the steps to submit them:
 
 1. **Create a New Branch**: Initiate your changes in a fresh branch. It's recommended to name the branch in a manner that signifies the changes you're implementing.
+3. **Add a changeset**: If you're updating any packages and want to ensure they're released, add a changeset to your branch by running `pnpm changeset` in the workspace root. For most changes you should use a **patch** bump.
```

Comment:
Lets have them always use a `patch` changeset - only we should be doing minor/major

Reply:
updated!